app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::BaseController < ApplicationController
+  before_action :require_admin
+
+  def require_admin
+    render file: 'public/404' unless current_admin?
```

Comment:
Thanks so much for building these authorizations. I think I saw something in the docs that looked cool - instead of rendering file 'public/404' we could also write render status: :not_found. Apparently more Rails magic! Not needed for this PR, but just a thought I had looking through this. :)

Reply:
I tried this and it didn't work :(